app/views/users/_edit_status_modal.html.haml

```diff
@@ -39,6 +39,6 @@
           = f.submit t('users.user.submit_button_label'), class: "btn btn-primary form-control",
                      id: 'editUserStatusSubmit'
 
-          %button{ type: 'button', class: 'btn btn-sm btn-default', data: { dismiss: 'modal' } }
+          %button{ type: 'button', class: 'btn btn-outline-secondary', data: { dismiss: 'modal' } }
```

Comment:
DRY:  seems like we could create a partial for the modal cancel button.

[should be a different PR.  If others agree, I'll write the story.]